src/server/transaction.cc

```diff
@@ -1370,19 +1370,10 @@ void Transaction::LogAutoJournalOnShard(EngineShard* shard, RunnableResult resul
     return;
 
   if (result.status != OpStatus::OK) {
-    // We log NOOP even for NO_AUTOJOURNAL commands because the non-success status could have been
```

Comment:
This used to be the same as `TriggerJournalWriteToSink` below, since `Op::NOOP` was ignored and only used to flush to sink